signer/signer.go

```diff
@@ -42,6 +42,10 @@ type RSACacheConfig struct {
 	// FetchTimeout is how long a consumer waits for the cache
 	// before generating its own key
 	FetchTimeout time.Duration
+
+	// MonitorSampleRate is how frequently the monitor reports the
+	// cache size and capacity
+	MonitorSampleRate time.Duration
```

Comment:
Since this has nothing to do with the `autograph-monitor`, I think we should use a different naming to avoid confusing. How about `StatsSamplingRate` ?